src/data/indicator_function.lean

```diff
@@ -93,6 +98,31 @@ lemma mem_range_indicator {r : β} {s : set α} {f : α → β} :
 by simp [indicator, ite_eq_iff, exists_or_distrib, eq_univ_iff_forall, and_comm, or_comm,
   @eq_comm _ r 0]
 
+/-- If `g` maps a second argument of 0 to 0, summing it for an
+indicator function over a possibly larger `finset` is the same as
+summing ‵g i (f i)` over the original `finset`. -/
```

Comment:
I think the actual statement is easier to read than this doc-string. Am I allowed to suggest leaving it out?

Reply:
Or maybe expand it to discuss the motivation for having this formulation with two functions (e.g. f gives some form of weights and g uses those weights in a multiplication or similar; my immediate use case has g being `(λ i wi, wi • (p i -ᵥ b : V))`)?

Reply:
Sure.

My complaint is largely that the phrasing "If `g` maps a second argument of 0 to 0 ..." is unreadable without reading the formal statement first.

Reply:
How does the rewritten version of the comment seem?

Reply:
Excellent! Thank you.